settings/cache.settings.php

```diff
@@ -18,38 +18,11 @@
 }
 
 /**
- * Example: Override required core defaults to use alternative cache backends.
- *
- * The bootstrap, discovery, and config bins use the chainedfast backend by
- * default and since the core service definition for these bins sets
- * the expire to CacheBackendInterface::CACHE_PERMANENT, these objects
- * are cached permanently and may result in stale configuration when
- * rebuilding the service container on deployments and cache rebuilds.
- * Uncomment the relevant lines for each bin below and configure the desired.
- *
- * $settings['cache']['bins']['bootstrap'] = 'cache.backend.null';
- * $settings['cache']['bins']['discovery'] = 'cache.backend.null';
- * $settings['cache']['bins']['config'] = 'cache.backend.null';
- *
- * @see https://www.drupal.org/node/2754947
- */
-
-/**
- * Use memcache as cache backend.
- *
- * Autoload memcache classes and service container in case module is not
- * installed. Avoids the need to patch core and allows for overriding the
- * default backend when installing Drupal.
- *
- * @see https://www.drupal.org/node/2766509
+ * Use memcache as cache backend if Acquia configuration is present.
  */
-if ((EnvironmentDetector::isAhEnv() || EnvironmentDetector::isCiEnv()) &&
-  // phpcs:ignore
-  array_key_exists('memcache', $settings) &&
-  // phpcs:ignore
-  array_key_exists('servers', $settings['memcache']) &&
+$repo_root = dirname(DRUPAL_ROOT);
+$memcacheSettingsFile = $repo_root . '/vendor/acquia/memcache-settings/memcache.settings.php';
```

Comment:
Should this still be wrapped in `EnvironmentDetector::isAhEnv()` or similar so it only includes on Acquia?

Reply:
The [upstream memcache settings file](https://docs.acquia.com/_downloads/cloud-memcache-d8.php) already includes this check in its preamble, so it would be redundant for BLT to check as well. Let me know if you think that would be insufficient.

I figured this would cause confusion, maybe I should have noted it in the code. Oh well :smile: